js/post/update_post.js

```diff
@@ -307,7 +303,7 @@ function fill300s() {
  * Autofills textboxes for 400 level courses. 
 **/
 function fill400s() {
-    'use-strict';
+    'use strict';
 
     var i = 0; 
```

Comment:
Ah, an extraneous variable that's not really necessary after refactoring. I have removed it. Thanks!